pkg/autoscaler/autoscaler.go

```diff
@@ -595,13 +612,14 @@ var _ = Describe("Autoscaler should", framework.LabelAutoscaler, Serial, func()
 			// we need to check against the number of ready nodes in the cluster since
 			// previous tests might have left nodes that are not ready or unschedulable.
 			By("Watching Cluster node count to ensure it remains consistent")
-			Consistently(func() (bool, error) {
+			Consistently(func() (int, error) {
 				nodes, err := framework.GetReadyAndSchedulableNodes(client)
-				return len(nodes) == caMaxNodesTotal, err
-			}, framework.WaitShort, pollingInterval).Should(BeTrue())
+				return len(nodes), err
+			}, framework.WaitShort, pollingInterval).Should(Equal(caMaxNodesTotal), "Cluster failed to stay consistent at %d nodes", caMaxNodesTotal)
 
 			By("Deleting the workload")
-			Expect(deleteObject(workload.Name, cleanupObjects[workload.Name])).Should(Succeed())
+			err := deleteObject(workload.Name, cleanupObjects[workload.Name])
+			Expect(err).Should(Succeed(), "Failed to delete scale-out workload %s", workload.Name)
```

Comment:
Please recombine these as they were before

Reply:
This one didn't get merged back together

Reply:
doh! i swear i triple checked the file before pushing, fml